tfhe/src/integer/server_key/radix/tests/modulus_switch_compression.rs

```diff
@@ -0,0 +1,61 @@
+use crate::integer::gen_keys_radix;
+use crate::integer::tests::create_parametrized_test;
+#[cfg(tarpaulin)]
+use crate::shortint::parameters::coverage_parameters::*;
+use crate::shortint::parameters::*;
+use rand::Rng;
+
+const NB_TESTS: usize = 10;
+
+create_parametrized_test!(modulus_switch_compression);
+create_parametrized_test!(modulus_switch_compression_signed);
+
+fn modulus_switch_compression<P>(param: P)
+where
+    P: Into<PBSParameters>,
+{
```

Comment:
did you check for the function executor pattern ? 

New tests are expected to use the function executor so that it's easy to test with different compute/hardware backends

Reply:
I talked about it with Thomas
It's not useful as long as the operation is only supported on the CPU